components/apimgt/org.wso2.carbon.apimgt.keymgt/src/main/java/org/wso2/carbon/apimgt/keymgt/validators/APIResourceScopeValidator.java

```diff
@@ -0,0 +1,243 @@
+package org.wso2.carbon.apimgt.keymgt.validators;
```

Comment:
license header is missing

Reply:
fixed in https://github.com/wso2/carbon-apimgt/pull/6232/commits/555a2ab2fa875057c6fadb316f7e426484108bd3